devtools/create_project/xcode.cpp

```diff
@@ -477,61 +486,62 @@ void XcodeProvider::setupFrameworksBuildPhase(const BuildSetup &setup) {
 
 	// Local libraries
 	if (CONTAINS_DEFINE(setup.defines, "USE_FAAD")) {
-		DEF_LOCALLIB_STATIC("libfaad");
+		DEF_LOCALXCFRAMEWORK("faad", projectOutputDirectory);
```

Comment:
Removing those `DEF_LOCALIB_STATIC` breaks the macOS build. Only a few libraries remain in the link list (those that you did not change as they are not used on iOS):

![image](https://user-images.githubusercontent.com/552105/229294577-4a5df852-dad1-4479-8329-73d4fd7ac7b8.png)
This results in the link phase failing.